### ISSUE TITLE
Support TNAuthorizationList extension

### ISSUE THREAD

**Issue by microshine** [PeculiarVentures/pv-certificates-viewer#92]
### ASN.1 specification
[RFC8226](https://datatracker.ietf.org/doc/html/rfc8226) Secure Telephone Identity Credentials: Certificates
### Certificate

```
-----BEGIN CERTIFICATE-----
MIHkMIGLAgEAMAAwWTATBgcqhkjOPQIBBggqhkjOPQMBBwNCAASYC4z23gzhlt9S
HiiAimnkFUQeU0JJyBo7a8s+s1xEwNXjTaFbDzKYaBrvW24IEaheFThkTNhP1qLD
/anUCSvIoCkwJwYJKoZIhvcNAQkOMRowGDAWBggrBgEFBQcBGgQKMAigBhYENzA5
SjAKBggqhkjOPQQDAgNIADBFAiBvFYdsjcu3TFwufUIoTyt9zVXpG3AX7h9FkKFr
YhksHgIhAKxGcynSlwcr4YT1FroBznU3No4/D0BScAppZmvP0euK
-----END CERTIFICATE-----
```

### View
Template:
```
Array [
<type>: <item>
]
```

Types:
SPC: `<string>`
Range: `start=<string> count=<number>`
One: `<string>`

**Comment by microshine** [comment]
Before we could start it I need to publish ASN.1 schema.  I'll do it soon

**Comment by donskov** [comment]
@microshine Any progress with it?

**Comment by microshine** [comment]
Published `@peculiar/asn1-rfc8226@2.0.39`

**Comment by donskov** [comment]
Added to the [web-component](https://understandingwebpki.com/?cert=MIHkMIGLAgEAMAAwWTATBgcqhkjOPQIBBggqhkjOPQMBBwNCAASYC4z23gzhlt9SHiiAimnkFUQeU0JJyBo7a8s%2Bs1xEwNXjTaFbDzKYaBrvW24IEaheFThkTNhP1qLD%2FanUCSvIoCkwJwYJKoZIhvcNAQkOMRowGDAWBggrBgEFBQcBGgQKMAigBhYENzA5SjAKBggqhkjOPQQDAgNIADBFAiBvFYdsjcu3TFwufUIoTyt9zVXpG3AX7h9FkKFrYhksHgIhAKxGcynSlwcr4YT1FroBznU3No4%2FD0BScAppZmvP0euK)
![image](https://user-images.githubusercontent.com/9417251/129553514-d824f772-1bc6-4b5f-96c7-e5a3a99e777b.png)
